### PR TITLE
Battery Level Fix

### PR DESCRIPTION
**Pull request by Lukas-Batema**
Working on the battery level fix...

### CODE REVIEW

**Comment by yashash1511** [comment]
is still show 10% lower battery % to me if charge > 80%

**Comment by Lukas-Batema** [comment]
Uhh... It is not making a deploy, @andrewstech...

**Comment by Lukas-Batema** [comment]
Wat

**Comment by andrewstech** [comment]
> Wat

Github actions are down.

**Comment by andrewstech** [comment]
https://www.githubstatus.com/incidents/vvthhf8gxt80

**Comment by Lukas-Batema** [comment]
> [githubstatus.com/incidents/vvthhf8gxt80](https://www.githubstatus.com/incidents/vvthhf8gxt80)

Hmm, I am not getting texts from GitHub, even though I signed up for SMS alerts...

**Comment by MalikIdreesHasanKhan** [comment]
It's back up.

**Comment by yashash1511** [comment]
image
dont remove it it is for browsers that don't support battery status api

File: src/components/taskbar/index.js [hunk 1]
```diff
@@ -52,13 +52,13 @@ const Taskbar = () => {
   };
 
   const changebatterystatus = (bt) => {
-    let level = bt.level * 100 || 100;
+    let level = bt.level * 100;
```
Comment (blueedgetechno) [inline src/components/taskbar/index.js]: that `|| 100` was for browsers that don't return battery percentage, It takes 100 as default.

**Comment by yashash1511** [comment]
fixed #293

**Pull request closed without merging**